test/examples_test.go

```diff
@@ -81,6 +82,10 @@ func substituteEnv(input []byte, namespace string) ([]byte, error) {
 	// the test namespace using `ko create -n`
 	output = defaultNamespaceRE.ReplaceAll(output, []byte("namespace: "+namespace))
 
+	// Replace image names to arch specific ones, where it's necessary
```

Comment:
Can you update the method's doc comment to mention this new replacement?

Reply:
done